polaris.shopify.com/src/components/TokenList/TokenList.tsx

```diff
@@ -446,6 +446,9 @@ function TokenPreview({name, value}: TokenPreviewProps) {
         {...previewDivAttributes}
         style={{
           display: 'flex',
+          background: 'var(--p-color-bg-app)',
+          borderRadius: 'var(--p-border-radius-200)',
+          padding: '1rem',
```

Comment:
Can use spacing tokens here:

```suggestion
          padding: 'var(--p-space-400)',
```